about.html

```diff
@@ -13,56 +13,75 @@
 
 <body>
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
+             <header>
+            <img class = "lambdaBlack" src = C:\Users\Mitch\Desktop\web-Sprint-Challenge-User-Interface-and-Git\img\lambda-black.png>
```

Comment:
For accessibility reasons, whenever you have a <img/> tag, its best practice to add an "alt" attribute to the tag with the description of the image, if the image doesn't  load the user will get a description of what the image is suppose to be and it allows programs, such as screen reader, to read out loud what the image is.